charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_poddisruptionbudget.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and (eq .Values.global.ha.enabled true) (or (eq .Values.global.scheduler.enabled true) (eq .Values.global.reminders.serviceName "scheduler")) }}
```

Comment:
Specifically looking for confirmation on this conditional for all chart files. I did: if ha enabled and either the scheduler is enabled or the reminder service name is scheduler.

Reply:
going to make this bare bones if scheduler is enabled, then add reminder condition later